builder/docker_build.py

```diff
@@ -0,0 +1,133 @@
+"""
+*****************************************************************
+Licensed Materials - Property of IBM
+(C) Copyright IBM Corp. 2020. All Rights Reserved.
+US Government Users Restricted Rights - Use, duplication or
+disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+*****************************************************************
+"""
+import os
+import datetime
+
+OPEN_CE_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+BUILD_IMAGE_PATH = os.path.join(OPEN_CE_PATH, "images/builder")
+LOCAL_FILES_PATH = os.path.join(os.path.dirname(OPEN_CE_PATH), "local_files")
+HOME_PATH = "/home/builder"
+
+REPO_NAME = "open-ce"
+IMAGE_NAME = "open-ce-builder"
+
+DOCKER_TOOL = "docker"
+
+def build_image():
```

Comment:
Does it make sense to have arguments for `REPO_NAME`, etc., and just have them default to the constants?
It could make it more flexible in the future.
If we do it here, we may want to do it for other functions in this file.

Reply:
It might in the future, but I would like to wait until there is a use case for it.